README.md

```diff
@@ -0,0 +1,52 @@
+SFML.Net - Simple and Fast Multimedia Library for .Net [![Build Status](https://travis-ci.org/SFML/SFML.Net.svg?branch=master)](https://travis-ci.org/SFML/SFML.Net)
+======================================================
+
+[SFML](sfml-dev.org) is a simple, fast, cross-platform and object-oriented multimedia API. It provides access to windowing,
+graphics, audio and network.
+It is originally written in C++, and this project is its official binding for .Net languages (C#, VB, ...).
+
+Authors
+-------
+
+Laurent Gomila - main developer (laurent@sfml-dev.org)
+Zachariah Brown - active maintainer (contact@zbrown.net)
```

Comment:
You should use a list here, otherwise it kind of butchers the output into a single line